dist/aphrodite.js

```diff
@@ -218,6 +218,9 @@ module.exports =
 	 */
 	var isUnitlessNumber = {
 	    animationIterationCount: true,
+	    borderImageOutset: true,
```

Comment:
Want correcting you. Mostly just a suggestion to the maintainers of this project :-)